sotodlib/hwp/g3thwp.py

```diff
@@ -621,7 +629,140 @@ def write_solution(self, solved, output=None):
             start_time += frame_length
 
         return
+    
+    def _write_empty_solution_h5(self, tod, output=None, h5_address=None):
+        logger.info('Writing empty solutions')
+        aman = sotodlib.core.AxisManager(tod.dets, tod.samps)
+        aman.wrap_new('timestamps', shape=('samps', ))
+        aman.wrap_new('hwp_angle_ver1', shape=('samps', ))
+        aman.wrap_new('hwp_angle_ver2', shape=('samps', ))
+        aman.wrap_new('stable', shape=('samps', ))
+        aman.wrap_new('locked', shape=('samps', ))
+        aman.wrap_new('hwp_rate', shape=('samps', ))
+        aman.wrap_new('eval', shape=('samps', ))
+
+        aman.timestamps = tod.timestamps
+        aman.stable = np.zeros(len(tod.timestamps))
```

Comment:
this line `aman.stable = np.zeros(len(tod.timestamps))` is identical to `aman.wrap_new('stable', shape=('samps', ))`. The first is more correct, so I would remove all the zeros you have down here. `wrap_new` has a `cls` argument that defaults to `np.zeros`.

We don't like doing `aman.field = new_thing` because that could break the axis tracking. If you have to, use `aman.field[:] = new_thing` to make sure the sizes are the same. But in these cases you do not need to reset these fields to zeros.